Woof/Woof/View/OwnerProfileView/OwnerProfileView.swift

```diff
@@ -0,0 +1,55 @@
+import SwiftUI
+
+/// View which could be used to display or edit information about Owner
+struct OwnerProfileView: View {
```

Comment:
Maybe better describe the view in a more abstract form?
```suggestion

/// View to display personal information about the owner.
struct OwnerProfileView: View {
```

Reply:
Agree.

Reply:
Agree, more abstract form better for the future functionality